src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -14,6 +14,13 @@
 
 namespace Microsoft.Cci
 {
+    internal sealed class PeWritingException : Exception
```

Comment:
Is there a reason to put this type into Microsoft.Cci namespace? It is not used in that namespace.